CoffeeChat/Controller/LoginViewController.swift

```diff
@@ -102,6 +102,17 @@ class LoginViewController: UIViewController {
             self.errorMessageAlertView.transform = .identity
         })
     }
+
+    private func createUser(idToken: String) {
+        NetworkManager.shared.createUser(idToken: idToken).observe { result in
+            switch result {
+            case .value(let response):
+                print(response)
```

Comment:
is this just a "placeholder" action for when networking is actually connected? if so, can be add a `TODO`, and if not & it's just for logging purposes, can we do a more formatted/detailed log?